cmd/capture.go

```diff
@@ -35,13 +40,29 @@ by passing the function name symbol and the binary args.
 `,
 	Example: "  harpoon -f main.doSomething ./command arg1 arg2 ...",
 	Run: func(cmd *cobra.Command, args []string) {
-		opts := captor.CaptureOptions{
+
+		functionSymbolList := strings.Split(functionSymbols, ",")
```

Comment:
I'm always worried when I see such code

And empty string will lead to a slice of 1 string with empty string in it.

So []string{""}

I would either check the length of functionSymbols before splitting, or I would skip in the for loop when strings.TrimSpace(item) == ""

I'm unsure if it can happen with your cost. But I cannot help myself to raise this to your attention

Reply:
Couldn't agree more with you. I'll add some checks to avoid that sometimes like so could happen. Thanks for raising the issue.

Reply:
Since the `--function` flag is marked as required, we should never face with an empty value in input.
So for now I would skip any control and add them in the future.